docker/README.md

```diff
@@ -66,14 +69,45 @@ Explanation of parts:
 * `-p 4190:8080` maps the internal algod REST API to local port 4190
 * `-e NETWORK=` can be set to any of the supported public networks.
 * `-e FAST_CATCHUP=` causes fast catchup to start shortly after launching the network.
-* `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis.
+* `-e TELEMETRY_NAME=` enables telemetry reporting to Algorand for network health analysis. The value of this variable takes precedence over the GUID in `/etc/logging.config`.
```

Comment:
Is it the GUID or the `Name`? (https://developer.algorand.org/docs/run-a-node/reference/telemetry-config/#configuration)

Reply:
I checked [`telemetryConfig.go`](https://github.com/algorand/go-algorand/blob/master/logging/telemetryConfig.go#L109) - This "name" suffixes the GUID.